e2e/tests/app.ts

```diff
@@ -0,0 +1,9 @@
+import {Page} from 'puppeteer';
+import GoogleLoginPage from '../app/google-login';
+
+
+export const signIn = async (page: Page) => {
+  await GoogleLoginPage.logIn(page);
+   // this element exists in DOM after user has logged in
+  await page.waitFor(() => document.querySelector('app-signed-in app-nav-bar clr-icon') !== null);
```

Comment:
do we need to depend on clr-icon? This seems maybe overly specific. Can we just wait for app-signed-in, for example?